typings/svg-properties.d.ts

```diff
@@ -282,13 +282,15 @@ type SVGPropertiesComplete = SingleOrArray<
   keyof SVGPropertiesCompleteSingle
 >
 
-export interface SVGPropertiesLossy {
+export interface SVGPropertiesLossyUnrestricted {
   [propertyName: string]:
     | string | number | SVGProperties | undefined
     | Array<SVGPropertiesCompleteSingle[keyof SVGPropertiesCompleteSingle]>
-    | SVGPropertiesLossy
+    | Omit<SVGPropertiesLossyUnrestricted, 'children'>
 }
 
+type SVGPropertiesLossy = Omit<SVGPropertiesLossyUnrestricted, 'children'>;
```

Comment:
The underlying issue here is that this indexer is not matching `{ children?: React.ReactNode }`.

So rather than omit from this indexer, we should be able to add `React.ReactNode` to the property union in the original `SVGPropertiesLossy`.

This strategy should also fix the tests which are no longer failing.